test/spec/tokenManager.js

```diff
@@ -307,13 +287,13 @@ define(function(require) {
       it('automatically refreshes a token by default', function(done) {
```

Comment:
It seems like we also need to test the inverse case, `does not refresh tokens if autoRefresh is false` ?

Reply:
This is captured with the modified test [`emits "expired" on existing tokens even when autoRefresh is disabled`](https://github.com/okta/okta-auth-js/pull/128/files/751733e36ae2efcc4388450e53000139396690fa#diff-dbfa077c685241b22c18a8a43ee006fbR395).

We're asserting two things with this test:
1. The token emits an expired event
2. When `autoRefresh` is set to false, the token getter will return undefined - indicating it **does not refresh**

Would it make more sense to break these tests out?

Reply:
I do prefer that the cases be explicit, so if it's easy to break out a new case that would be ideal.

Reply:
Done. Created a new [`returns undefined for a token that has expired when autoRefresh is disabled`](https://github.com/okta/okta-auth-js/pull/128/commits/557067115db75864b355c980f026a550b8bb1746#diff-dbfa077c685241b22c18a8a43ee006fbR397) test.